### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by eirsep**
Release Notes for odfe release 1.13.0.0
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by anshul291995 — APPROVED** [review]

**Review by rramachand21 — APPROVED** [review]

File: release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+### Maintenance
+* Supports Elasticsearch 7.10.2 ([#32](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/32))
+
+
+### Features
+* Listener framework for asynchronous search ([#2](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/2))
+* Service layer and Transport Handlers ([#8](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/8))
+* Rest Layer and Async Search Cleanup Management ([#9](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/9))
+* Integrates security with asynchronous search ([#11](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/11))
```
Comment (Bukhtawar) [inline release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md:12]: I guess we should talk about the Cx and usage of the APIs instead

File: release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## 2021-02-03 Version 1.13.0.0
+Initial launch of Asynchronous Search on opendistro-for-elasticsearch
```
Comment (rramachand21) [inline release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md:2]: can we put a brief explanantion of what it does and we can also link a reference to detailed RFC/Document

File: release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md (hunk 1 above)
Comment (eirsep) [reply release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md:18]: added a brief intro about plugin.

File: release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md (hunk 2 above)
Comment (eirsep) [reply release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md]: added the same.

File: release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+The asynchronous search APIs let you asynchronously execute a search request, monitor its progress, and retrieve partial results as they become available.
+
+### Maintenance
+* Supports Elasticsearch 7.10.2 ([#32](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/32))
+
+
+### Features
+* Listener framework for asynchronous search ([#2](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/2))
+* Service layer and Transport Handlers ([#8](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/8))
+* Rest Layer and Async Search Cleanup Management ([#9](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/9))
```
Comment (Bukhtawar) [inline release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md:17]: asynchronous

File: src/test/java/com/amazon/opendistroforelasticsearch/search/asynchronous/integTests/AsynchronousSearchRejectionIT.java [hunk 4]
```diff
@@ -96,15 +97,19 @@ public void testSimulatedSearchRejectionLoad() throws Throwable {
...
                                 @Override
                                 public void onResponse(AsynchronousSearchResponse asynchronousSearchResponse) {
-                                    if (asynchronousSearchResponse.getSearchResponse() == null) {
+                                    if (asynchronousSearchResponse.getSearchResponse() != null) {
+                                        responses.add(asynchronousSearchResponse.getSearchResponse());
+                                    } else if(asynchronousSearchResponse.getError() != null){
                                         responses.add(asynchronousSearchResponse.getError());
                                     } else {
-                                        responses.add(asynchronousSearchResponse.getSearchResponse());
+                                        logger.warn("No error or response received : " + asynchronousSearchResponse);
```
Comment (Bukhtawar) [inline src/test/java/com/amazon/opendistroforelasticsearch/search/asynchronous/integTests/AsynchronousSearchRejectionIT.java:112]: Why warn how does it help?

File: src/test/java/com/amazon/opendistroforelasticsearch/search/asynchronous/integTests/AsynchronousSearchRejectionIT.java [hunk 5]
```diff
@@ -109,7 +109,7 @@ public void onResponse(AsynchronousSearchResponse asynchronousSearchResponse) {
                                     } else if(asynchronousSearchResponse.getError() != null){
                                         responses.add(asynchronousSearchResponse.getError());
                                     } else {
-                                        logger.warn("No error or response received : " + asynchronousSearchResponse);
+                                        logger.debug("No error or response received : " + asynchronousSearchResponse);
```
Comment (Bukhtawar) [inline src/test/java/com/amazon/opendistroforelasticsearch/search/asynchronous/integTests/AsynchronousSearchRejectionIT.java]: why log?

Comment (eirsep) [reply src/test/java/com/amazon/opendistroforelasticsearch/search/asynchronous/integTests/AsynchronousSearchRejectionIT.java]: hmm. agreed. removing this log message.

**Review by Bukhtawar — APPROVED** [review]
Just confirm is we are using the same template as that of any new plugin

**Review by anshul291995 — APPROVED** [review]

**Pull request merged**